docs/tempo/website/operations/consistent_hash_ring.md

```diff
@@ -25,6 +25,12 @@ This ring is only used when "global" rate limits are used. The distributors use
 
 This ring is used by the distributors to load balance traffic into the ingesters. When spans are received the trace id is hashed and they are sent to the appropriate ingesters based on token ownership in the ring. Queriers also use this ring to find the ingesters for querying recent traces.
 
+#### Metrics-generator
+**Participants:** Metrics-generators  
+**Used by:** Distributors
+
+This ring is used by distributors to load balance traffic to the metrics-generators. When spans are received, the trace ID is hashed, and the traces are sent to the appropriate metrics-generators based on token ownership in the ring.
```

Comment:
Should we mention what the hash is?

Reply:
I think that's an implementation detail of the ring in dskit. How a hash ring works is explained here: https://cortexmetrics.io/docs/architecture/#the-hash-ring (this link is also on the top of this page in the docs)